README.md

```diff
@@ -63,7 +69,7 @@ Required Environment Variables:
   - CVS has a maximum of 25, vaccine-watch will use 25 for CVS if you set this higher.
 - `LATITUDE`: Latitude of the location to check (e.g. `39.1040382`)
 - `LONGITUDE`: Latitude of the location to check (e.g. `-94.5701803`)
-- `STATES`: JSON: Abbreviations of which states are in radius of your location (e.g. `["MO", "KS"]`)
+- `STATES`: JSON: Abbreviations of which states are in radius of your location (e.g. `["MO", "KS", "IA"]`)
```

Comment:
Same here, let's keep the example to just two states

Reply:
change implemented.